src/components/main/note/EditDialog.tsx

```diff
@@ -94,14 +101,21 @@ const EditDialog: FC<EditDialogProps> = ({
           color="error"
           onClick={onDelete}
           endIcon={<DeleteOutlinedIcon />}
+          data-cy={DELETE_BUTTON_IN_DIALOG_CY} // TODO: fix this. It doesn't work for testing...
```

Comment:
Yes, since this is a "graasp" component button which wraps the actual MUIButton the `data-cy` prop is not passed to the dom. But we exposed a `dataCy` props that you can use exactly the same ! See under the controls section [here](https://graasp.github.io/graasp-ui/?path=/story/buttons-button--primary)

Reply:
Thanks, I did not know that and did not think about it!